roles/icinga_plugins/molecule/default/converge.yml

```diff
@@ -0,0 +1,159 @@
+---
+- name: Converge
+  hosts: all
+
+  pre_tasks:
```

Comment:
Could these go into the `prepare` stage? The prepare stage only gets executed once and does not have to be idempotent.

Reply:
Also isn't there a role to do all this?

Reply:
> Could these go into the `prepare` stage? The prepare stage only gets executed once and does not have to be idempotent.

done

Reply:
> Also isn't there a role to do all this?

Normally icinga_agent role will install the icinga2 package, but we dont want to mix the tests for these roles.